### ISSUE TITLE
'RequestContext' object has no attribute 'META' in Django 1.10

### ISSUE THREAD

**Issue by dlwxxxdlw** [HonzaKral/django-threadedcomments#81]
Environment:

Request Method: GET
Request URL: http://localhost:8000/sblog/blog/3/

Django Version: 1.10.1
Python Version: 2.7.10
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'sblog',
 'threadedcomments',
 'django.contrib.sites',
 'django_comments']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']

Template error:
In template /Users/dlw/Documents/DLW/Django/blogDemo/sblog/templates/sblog/blog_show.html, error at line 46
   'RequestContext' object has no attribute 'META'   36 :

```
    {% load threadedcomments_tags %}
    37 :                     {% get_comment_count for blog as comment_count %}
    38 :                     <a href="#cmt" title="comment"><i class="icon-comment offset1"></i>{{ comment_count }} </a>
    39 :                 </div>
    40 :             </div>
    41 :         </section>
    42 :     </article>
    43 : </div>
    44 : {% block mycomments %}
    45 :     <h2>Comments for {{ blog }}</h2>
    46 :     {% render_comment_list for blog %}            ##Here is the error
    47 :     {% render_comment_form for blog %}
    48 : {% endblock %}
    49 : {% endblock %}
    50 : 
```

Traceback:

File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py" in inner
1.             response = get_response(request)

File "/Library/Python/2.7/site-packages/django/core/handlers/base.py" in _get_response
1.                 response = self.process_exception_by_middleware(e, request)

File "/Library/Python/2.7/site-packages/django/core/handlers/base.py" in _get_response
1.                 response = wrapped_callback(request, _callback_args, *_callback_kwargs)

File "/Users/dlw/Documents/DLW/Django/blogDemo/sblog/views.py" in blog_show
1.     return render(request,"sblog/blog_show.html",{"blog":blog})

File "/Library/Python/2.7/site-packages/django/shortcuts.py" in render
1.     content = loader.render_to_string(template_name, context, request, using=using)

File "/Library/Python/2.7/site-packages/django/template/loader.py" in render_to_string
1.     return template.render(context, request)

File "/Library/Python/2.7/site-packages/django/template/backends/django.py" in render
1.             return self.template.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                     return self._render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in _render
1.         return self.nodelist.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 bit = node.render_annotated(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render_annotated
1.             return self.render(context)

File "/Library/Python/2.7/site-packages/django/template/loader_tags.py" in render
1.         return compiled_parent._render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in _render
1.         return self.nodelist.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 bit = node.render_annotated(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render_annotated
1.             return self.render(context)

File "/Library/Python/2.7/site-packages/django/template/loader_tags.py" in render
1.         return compiled_parent._render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in _render
1.         return self.nodelist.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 bit = node.render_annotated(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render_annotated
1.             return self.render(context)

File "/Library/Python/2.7/site-packages/django/template/loader_tags.py" in render
1.                 result = block.nodelist.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 bit = node.render_annotated(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render_annotated
1.             return self.render(context)

File "/Library/Python/2.7/site-packages/django/template/loader_tags.py" in render
1.                 result = block.nodelist.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 bit = node.render_annotated(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render_annotated
1.             return self.render(context)

File "/Library/Python/2.7/site-packages/django/template/loader_tags.py" in render
1.                 result = block.nodelist.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 bit = node.render_annotated(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render_annotated
1.             return self.render(context)

File "/Library/Python/2.7/site-packages/django/template/loader_tags.py" in render
1.                 result = block.nodelist.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 bit = node.render_annotated(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render_annotated
1.             return self.render(context)

File "/Library/Python/2.7/site-packages/threadedcomments/templatetags/threadedcomments_tags.py" in render
1.             }, context)

File "/Library/Python/2.7/site-packages/django/template/loader.py" in render_to_string
1.     return template.render(context, request)

File "/Library/Python/2.7/site-packages/django/template/backends/django.py" in render
1.             return self.template.render(context)

File "/Library/Python/2.7/site-packages/django/template/base.py" in render
1.                 with context.bind_template(self):

File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py" in **enter**
1.             return self.gen.next()

File "/Library/Python/2.7/site-packages/django/template/context.py" in bind_template
1.             updates.update(processor(self.request))

File "/Library/Python/2.7/site-packages/django/template/context_processors.py" in debug
1.     if settings.DEBUG and request.META.get('REMOTE_ADDR') in settings.INTERNAL_IPS:

Exception Type: AttributeError at /sblog/blog/3/
Exception Value: 'RequestContext' object has no attribute 'META

**Comment by dlwxxxdlw** [comment]
Now I change the Django-1.10 to Django-1.9.7, it works for me.

**Comment by vdboor** [comment]
This issue is now fixed; the new 1.1 release has Django 1.10 support
